server/service/src/document/document_service.rs

```diff
@@ -103,7 +103,10 @@ pub trait DocumentServiceTrait: Sync + Send {
                 }
 
                 match insert_document(con, store_id, doc) {
-                    Ok(doc) => Ok(doc),
+                    Ok(doc) => {
+                        document_updated(con, store_id, &doc)?;
```

Comment:
I think we should eventually use change log for this. Perhaps even in sync processor (since we would need to do this when documents are incoming), here we could just trigger the processor.

Reply:
good idea I added an issue: https://github.com/openmsupply/open-msupply/issues/76

Reply:
When i was reading another PR (inserting on patient), i thought it might be too hard to use change log (since we want to return the patient)